_data/events.yml

```
@@ -55,7 +55,7 @@ scala_lunches:
 
     - date: 14 March 2024
       time: "12:00pm"
-      place: BC 410, EPFL
+      place: **BC 229**, EPFL
```

```suggestion
      place: "**BC 229**, EPFL"
```

these quotes are needed or else the events can't be loaded from the file